services/121-service/seed-data/instance/instance-pilot-nl-2.json

```diff
@@ -10,20 +10,20 @@
-    "in": "Terakhir di update pada tgl: 16-11-2021<br><br>Kami,<strong>The Netherlands Red Cross</strong>butuh menanyakan informasi di bawah untuk dapat memberikan bantuan voucher supermarket<br><br>Kami akan menanyakan dari anda <ul><li>Nama depan, nama belakang </li><li>Password ( 4 angka yang harus anda pilih) </li><li>nomor HP untuk SMS</li><li>nomor HP untuk WhatsApp</li></ul><br>Informasi ini akan disimpan secara aman di komputer Red Cross dan hanya akan di-share dengan organisasi anda yang telah membagi nomor HP anda untuk menjamin proses yang aman.<br>Tergantung pada posisi anda, salah satu dari organisasi ini adalah <ul><li>Stap Verder</li><li>Filipino LBGT Europe</li><li>Opportunities in Amsterdam</li></ul><br><a href='https://www.twilio.com/legal/privacy' target='_blank' rel='noopener'>Twilio</a>yang akan menerima nomor HP anda agar kami bisa mengirim ke anda <ul>Sms untuk memberi informasi tentang program bantuan ini<li>pesan WA untuk mengirim voucher supermarket (apabila registrasi anda berhasil)</li></ul><br>Anda bisa meninggalkan program bantuan ini dan menghapus informasi pribadi anda kapanpun.<br>Bagaimanapun juga kami akan menghapus informasi pribadi anda saat proyek ini berakhir.<br><br>Apabila anda ada pertanyaan silahkan hubungi kami melalui WA di<a target='_blank' rel='noopener' href='https://wa.me/31614458781'> +31 6 14 45 87 81</a>"
+    "en": "Last updated on: 16-11-2021   <br><br>   We, <strong>The Netherlands Red Cross (Rode Kruis)</strong> need to ask you the information below in order to provide the supermarket voucher support.   <br><br>   We will ask you your: <ul> <li> First name, last name </li> <li> Phone number for SMS </li> <li> Phone number for WhatsApp </li> </ul>   <br>   This information will be stored securely on a Rode Kruis computer and shared only with the organization that gave us your phone number, to ensure a fair process.   <br>   This is, depending on your situation, one of the following organizations: <ul><li> Stap Verder </li> <li> Filipino LGBT Europe </li> <li> Opportunities in Amsterdam </li> </ul>   <br>   <a href='https://www.twilio.com/legal/privacy' target='_blank' rel='noopener'>Twilio</a> will receive your phone number so that we can send you: <ul> <li> SMS's to inform you about the aid program </li> <li> WhatsApp messages to send supermarket vouchers (if your registration is successful) </li> </ul>   <br>   You can leave the aid program and delete your personal information at any time. <br> We will delete your personal information regardless at the end of the project.   <br><br>   If you have any questions, please contact us via WhatsApp on: <a target='_blank' rel='noopener' href='https://wa.me/31614458781'>+31 6 14 45 87 81</a>",
```

Comment:
```suggestion
    "en": "Last updated on: 11-01-2022   <br><br>   We, <strong>The Netherlands Red Cross (Rode Kruis)</strong> need to ask you the information below in order to provide the supermarket voucher support.   <br><br>   We will ask you your: <ul> <li> First name, last name </li> <li> Phone number for SMS </li> <li> Phone number for WhatsApp </li> </ul>   <br>   This information will be stored securely on a Rode Kruis computer and shared only with the organization that gave us your phone number, to ensure a fair process.   <br>   This is, depending on your situation, one of the following organizations: <ul><li> Stap Verder </li> <li> Filipino LGBT Europe </li> <li> Opportunities in Amsterdam </li> </ul>   <br>   <a href='https://www.twilio.com/legal/privacy' target='_blank' rel='noopener'>Twilio</a> will receive your phone number so that we can send you: <ul> <li> SMS's to inform you about the aid program </li> <li> WhatsApp messages to send supermarket vouchers (if your registration is successful) </li> </ul>   <br>   You can leave the aid program and delete your personal information at any time. <br> We will delete your personal information regardless at the end of the project.   <br><br>   If you have any questions, please contact us via WhatsApp on: <a target='_blank' rel='noopener' href='https://wa.me/31614458781'>+31 6 14 45 87 81</a>",
```